src/glyph/BBox.js

```diff
@@ -45,20 +45,24 @@ export default class BBox {
   }
 
   addPoint(x, y) {
-    if (x < this.minX) {
-      this.minX = x;
-    }
+    if (Math.abs(x) != Infinity) {
```

Comment:
this is a coercive comparison. While Infinity coerces less than most values, `!==` is still a good idea.